backend/app/controllers/spree/admin/shipping_methods_controller.rb

```diff
@@ -16,6 +16,14 @@ def destroy
         end
       end
 
+      def permitted_resource_params
```

Comment:
@KacperMekarski we should use https://github.com/spree/spree/blob/master/core/lib/spree/permitted_attributes.rb for this